packages/ionic/src/commands/package/package.ts

```diff
@@ -0,0 +1,344 @@
+        'android debug --commit=2345cd3305a1cf94de34e93b73a932f25baac77c',
+        'ios development --security-profile="iOS Security Profile Name" --target-platform="iOS - Xcode 9"',
+        'ios development --security-profile="iOS Security Profile Name" --build-file-name=my_custom_file_name.ipa',
+      ],
+      inputs: [
+        {
+          name: 'platform',
+          summary: `The platform to package (${PLATFORMS.map(v => chalk.green(v)).join(', ')})`,
+          validators: [validators.required, contains(PLATFORMS, {})],
+        },
+        {
+          name: 'type',
+          summary: `The build type (${BUILD_TYPES.map(v => chalk.green(v)).join(', ')})`,
+          validators: [validators.required, contains(BUILD_TYPES, {})],
+        },
+      ],
+      options: [
+        {
+          name: 'security-profile',
+          summary: 'Security profile',
+          type: String,
+          spec: { value: 'name' },
+        },
+        {
+          name: 'environment',
+          summary: 'The group of environment variables exposed to your build',
+          type: String,
+          spec: { value: 'name' },
+        },
+        {
+          name: 'native-config',
+          summary: 'The group of native config variables exposed to your build',
+          type: String,
+          spec: { value: 'name' },
+        },
+        {
+          name: 'commit',
+          summary: 'Commit (defaults to HEAD)',
+          type: String,
+          groups: [OptionGroup.Advanced],
+          spec: { value: 'sha1' },
+        },
+        {
+          name: 'target-platform',
+          summary: 'Target platform',
+          type: String,
+          groups: [OptionGroup.Advanced],
+          spec: { value: 'name' },
+        },
+        {
+          name: 'build-file-name',
+          summary: 'The name for the downloaded build file',
+          type: String,
+          groups: [OptionGroup.Advanced],
+          spec: { value: 'name' },
+        },
+      ],
+    };
+  }
+
+  async preRun(inputs: CommandLineInputs, options: CommandLineOptions): Promise<void> {
+    if (!inputs[0]) {
+      const platformInput = await this.env.prompt({
+        type: 'input',
+        name: 'platform',
+        message: `Platform to package (${PLATFORMS.map(v => chalk.green(v)).join(', ')}):`,
+        validate: v => validators.required(v) && contains(PLATFORMS, {})(v),
+      });
+
+      inputs[0] = platformInput;
+    }
+
+    const buildTypes = inputs[0] === 'ios' ? IOS_BUILD_TYPES : ANDROID_BUILD_TYPES;
+
+    // validate that the build type is valid for the platform
+    let reenterBuilType = false;
+    if (inputs[1] && !buildTypes.includes(inputs[1])) {
+      reenterBuilType = true;
+      this.env.log.warn(`Build type ${chalk.bold(inputs[1])} incompatible for ${chalk.bold(inputs[0])}`);
+      this.env.log.nl();
+    }
+
+    if (!inputs[1] || reenterBuilType) {
+      const typeInput = await this.env.prompt({
+        type: 'input',
```

Comment:
You can use a list type https://github.com/SBoudrias/Inquirer.js/#list---type-list